src/coprocessor/codec/error.rs

```diff
@@ -11,15 +11,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::error::Error as StdError;
 use std::io;
 use std::str::Utf8Error;
 use std::string::FromUtf8Error;
 use std::{error, str};
 use tipb::expression::ScalarFuncSig;
 use tipb::select;
+
+use regex::Error as RegexpError;
```

Comment:
put a blank line between this and util, because they belong to different import groups

Reply:
OK, thanks